autotst/speciesTest.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+
+from autotst.species import Species, Conformer
+import rdkit
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from rdkit.Chem.rdchem import Mol
+import autotst
+import ase
+from ase import Atom, Atoms
+import rmgpy
+from rmgpy.molecule import Molecule as RMGMolecule
+from rmgpy.species import Species as RMGSpecies
+from autotst.geometry import Bond, Angle, Torsion, CisTrans, ChiralCenter
+
+class TestConformer(unittest.TestCase):
+    def setUp(self):
+        self.conformer = Conformer(smiles='CC')
+    def test_rmg_molecules(self):
+        self.assertIsInstance(self.conformer.rmg_molecule,RMGMolecule)
+    def test_rdkit_mol(self):
+        autotst_rdkit = self.conformer.get_rdkit_mol()
+        self.assertIsInstance(autotst_rdkit,Mol)
+    def test_ase_mol(self):
+        autotst_ase_mol = self.conformer.get_ase_mol()
+        self.assertIsInstance(autotst_ase_mol,Atoms)
+    def test_get_molecules(self):
+        autotst_rdkit, autotst_ase_mol = self.conformer.get_molecules()
+        self.assertIsInstance(autotst_rdkit,Mol)
+        self.assertIsInstance(autotst_ase_mol,Atoms)
```

Comment:
Should add the following tests:
- assert that all three molecules have the same XYZ coordinates 
- assert that changing the XYZ coordinates and running `update_coords` works